content/en/docs/07/labs/71.md

```diff
@@ -129,6 +129,11 @@ Filesystem                Size      Used Available Use% Mounted on
 /dev/sda1                10.0G      8.4G      1.6G  84% /prometheus
 ```
 
+{{% alert title="Note" color="primary" %}}
+We use Minkube, which for demonstration purposes uses the `/dev/sda1` disk of your virtual machine as the storage backend for all Persistent Volumes. Therefore, you will always see a size of `10.0G` when checking from inside a container. If you use an appropriate storage backend, the size inside the container will correspond to the size of your Persistent Volume.
```

Comment:
maybe we should not specify the actual size in GB here in case we need to change it in the future. Something like

```suggestion
We use Minkube, which for demonstration purposes uses the `/dev/sda1` disk of your virtual machine as the storage backend for all Persistent Volumes. Therefore, you will always see the size of `/dev/sda1` when checking from inside a container. If you use an appropriate storage backend, the size inside the container will correspond to the size of your Persistent Volume.